app/src/main/resources/application.yml

```diff
@@ -33,6 +35,15 @@ spring:
   main:
     allow-bean-definition-overriding: true
 
+cas-initializr:
+  supported-versions:
```

Comment:
This is really good.

One other suggestion here would be to reserve a placeholder for the version that is defined in gradle.properties and use resource-filtering. Idea is, if the latest version is updated in gradle.properties, that should automatically be reflected in the application.yml file as a supported version without manual work. I think this resource-filtering is done in other areas in the same file for CAS and Boot versions.